xcat-inventory/xcclient/xcatd/client/xcat_msg.py

```diff
@@ -426,7 +426,18 @@ def parse(self):
                         to_return.success_nodes.append(node.name)
 
                     elem.clear()
-
+                elif elem.tag == 'info':
+                    node = self._rpower_rec_parser.parse(elem)
```

Comment:
this method is only used to parse the node section, not use it to parse the `info` section